recipes-devtools/opn-tools/python3-opn-tools.bb

```diff
@@ -1,8 +1,8 @@
 LICENSE = "CLOSED"
 
-SRC_URI = "git://git@github.com/HULKs/opn-tools.git;branch=main;protocol=https"
+SRC_URI = "git://git@github.com/HULKs/opn-tools.git;branch=backeBackeKuchen;protocol=https"
```

Comment:
I think, this can be updated, as it is merged now

Reply:
Done